nvme-wrap.c

```diff
@@ -208,12 +208,15 @@ int nvme_cli_get_log_fw_slot(struct nvme_dev *dev, bool rae,
 	return do_admin_op(get_log_fw_slot, dev, rae, fw_log);
 }
 
-int nvme_cli_get_log_changed_ns_list(struct nvme_dev *dev, bool rae,
-				     struct nvme_ns_list *ns_log)
+int nvme_cli_get_log_changed_ns_list(struct nvme_dev *dev, bool rae, __u32 len,
+				     struct nvme_ns_list *ns_log, bool alloc)
 {
+	if (alloc)
+		return nvme_get_log_changed_alloc_ns_list(dev->direct.fd, rae, len, ns_log);
```

Comment:
I suppose the `do_admin_op` wrapper should be used here as well. Do we miss an API update in libnvme-mi?

Reply:
Yes sorry for the missing so will update libnvme-mi to add APIs later. Thank you.

Reply:
Please introduce a new wrapper function for `nvme_get_log_changed_alloc_ns_list`. These function here should really only switch between the standard API and MI API interfaces. There should be no logic besides this. Note, these wrappers will go away when we start with libnvme 2.x hopefully soonish.

Reply:
Understood and noted then fixed the changes as so. Thank you.